.circleci/integration-tests/Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN cp nuke-config.yml  ${AIRFLOW_HOME}/dags/
 USER astro
 
 # Deploy from astro runtime image
-FROM quay.io/astronomer/astro-runtime:4.2.4-base as astro-cloud
+FROM quay.io/astronomer/astro-runtime:4.2.6-base as astro-cloud
```

Comment:
runtime should be `quay.io/astronomer/astro-runtime:5.0.0`

Reply:
airflow image also needs change in first line `FROM quay.io/astronomer/ap-airflow:2.2.5 as staging` to `FROM quay.io/astronomer/ap-airflow:2.3.0 as staging` ?

Reply:
Have updated tag for astro-runtime image. For staging, we can create another issue and PR. The scope of the issue to move integration tests to Astro Cloud.,